src/harness/testcases/spectrum_inquiry_testcase.py

```diff
@@ -30,6 +30,23 @@ def tearDown(self):
     pass
 
   @winnforum_testcase
+  def test_WINFF_FT_S_SIQ_8(self):
+    """Send Spectrum Inquiry with missing cbsdId field.
+
+    The response should be MISSING_PARAM, code 102
+    """
+    # Send Spectrum Inquiry request. Note: registration is not relevant for this
+    # case.
+    spectrum_inquiry_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    self.assertFalse('cbsdId' in spectrum_inquiry_0)
+    request = {'spectrumInquiryRequest': [spectrum_inquiry_0]}
+    # Check Spectrum Inquiry Response
+    response = self._sas.SpectrumInquiry(request)['spectrumInquiryResponse'][0]
+    self.assertFalse('cbsdId' in response)
+    self.assertEqual(response['response']['responseCode'], 102)
+
```

Comment:
I went back to the document. It MAY be possible to identify the device based on the device certificate. And the test case mentions "Make sure the CBSD has registered with the SAS and obtained a cbsdId". I am having second thoughts about skipping the registration, as it materially changes the test instructions.